scripts/python/db_sync_transactions_dump.py

```diff
@@ -0,0 +1,56 @@
+import asyncio
+import asyncpg
+from collections import namedtuple
+import json
+import base64
+
+QUERY = """
+WITH
+meta_table AS (
+    select tx_id, json AS metadata from tx_metadata where key = '61284'
+    ) ,
+sig_table AS (
+    select tx_id, json AS signature from tx_metadata where key = '61285'
+    )
+SELECT tx.hash, tx_id, metadata, signature, block_index
+FROM meta_table
+    INNER JOIN tx ON tx.id = meta_table.tx_id
+    INNER JOIN sig_table USING(tx_id)
+    INNER JOIN block ON block.id = tx.block_id
+ORDER BY metadata -> '4' ASC;
+"""
```

Comment:
This comes from a [conversation ](https://input-output-rnd.slack.com/archives/CC0CLNEH1/p1635432012025000) with Samuel. 
Is it cryptic? yeah, it is.